src/test/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("The " + getName() + " began its work.");
+
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The " + getName() + " has finished its work.");
```

Comment:
Similarly, the message in the 'stopWork' method should clearly indicate that the 'Truck' has stopped working. For instance, 'Truck ' + getName() + ' finished its work.'